test/test_images/runtime/handlers/mount.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2019 The Knative Authors
+ Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+     http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package handlers
+
+import (
+	"bufio"
+	"os"
+	"strings"
+
+	"github.com/knative/serving/test/types"
+)
+
+func mounts() []*types.Mount {
+	var mounts []*types.Mount
```

Comment:
perhaps, move this down right before the `for` loop?

Reply:
moved